starport_template/lib/utils/cosmos_transaction_history.dart

```diff
@@ -0,0 +1,47 @@
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+import 'package:starport_template/entities/amount.dart';
+import 'package:starport_template/entities/denom.dart';
+import 'package:starport_template/entities/transaction.dart';
+import 'package:starport_template/model/tx_response_json.dart';
+import 'package:starport_template/utils/base_env.dart';
+
+class CosmosTransactionHistory {
```

Comment:
this is not an entity holding the history of transactions, its more of a repository or `HistoryLoader`